samples/AWSTaskQueueSamples/Greetings/Greetings.csproj

```diff
@@ -8,6 +8,7 @@
     <ProjectReference Include="..\..\..\src\Paramore.Brighter\Paramore.Brighter.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.4" />
```

Comment:
Did you mean to commit all these DependencyInjection packages?

Reply:
I updated the package dependencies as part of pulling RMQ 6 in, and these got picked up

Reply:
I have also updated the merge to reflect changes to packages that have since occurred via dependabot

Reply:
We can roll that back if it really bothers you, but it would save effort if we just accept we need to upgrade it

Reply:
Another way to look at this.

This PR fixed package upgrades that we normally do via Dependabot. One couldn't be done by merging as it contained breaking changes (RMQ), one was a deprecation issue, which again Dependabot won't resolve; and the other was an effort to make sure that all the packages were up to date, before updating any packages manually

Reply:
Updated scope of PR

Reply:
dependabot annoys me it's good for notifications, it's the merges are a mess. 

I'll may have to remove when I merge in my branch as they not needed.

otherwise let's rebase this in